src/schemas/index.ts

```diff
@@ -7,6 +7,8 @@ import * as rfqMakerConfigSchema from './rfq_maker_config.json';
 import * as rfqMakerConfigListSchema from './rfq_maker_config_list.json';
 import * as slippageModelFileSchema from './slippage_model_file_schema.json';
 import * as slippageModelSchema from './slippage_model_schema.json';
+import * as sraofferAddLiquiditySchema from './sra_offer_add_liquidity_schema.json';
```

Comment:
use capital "O" after "sra" in line with line 11

Reply:
Sorry. I will fix this issue.